taxi_service/urls.py

```diff
@@ -15,7 +15,9 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.urls.conf import include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("taxi/", include("taxi.urls", namespace="taxi")),
```

Comment:
check this requirement again
![image](https://github.com/user-attachments/assets/1f2a15a5-9f82-45f3-8f28-141076e47891)